doc/classes/TreeItem.xml

```diff
@@ -411,6 +418,20 @@
 				If [code]true[/code], the column [code]column[/code] is checked.
 			</description>
 		</method>
+		<method name="set_collapsed_recursive">
+			<return type="void">
+			</return>
+			<argument index="0" name="enable" type="bool">
+			</argument>
+			<argument index="1" name="ignore_active" type="bool">
+			</argument>
+			<argument index="2" name="ignore_self" type="bool">
+			</argument>
+			<description>
+				If [code]enable[/code] is [code]true[/code] collapses, else uncollapses, this TreeItem and all its descendants recursively, ignoring any active TreeItem if [code]ignore_active[/code] is [code]true[/code].
+				If [code]ignore_self[/code] is [code]true[/code], only affects descendants.
```

Comment:
Only affects descendants when [code]ignore_self[/code] is [code]true[/code].

Reply:
I think this could lead to confusion too. It could be interpreted as "descendants are not affected if ignore self is false".

How about:
> Only descendants are affected when [code]ignore_self[/code] is [code]true[/code].